### ISSUE TITLE
Scaling up DINO to larger model size

### ISSUE THREAD

**Issue by shallowtoil** [facebookresearch/dino#138]
Hi @mathildecaron31, I'm recently considering scaling DINO to a larger model size, e.g., ViT-L/16. I used the almost same parameters as ViT-B/16 and pre-train DINO for 400 epochs but the k-NN and linear probing accuracy are ~1% and ~2% worse than the base-size model respectively. Do you have any related experience with that? Thanks for your help!

**Comment by woctezuma** [comment]
Related: #13

**Comment by shallowtoil** [comment]
> Since you specifically mention the k-NN task, this could be linked to:
> 
> * [Rationale behind DeiT-S/8 being better than ViT-B/8 at k-NN? #13](https://github.com/facebookresearch/dino/issues/13)
> 
> ![Excerpt from Table 2](https://camo.githubusercontent.com/d7a876e938ff41a5bdfd68f45bfc86d54eb12ec98caf6c7c9566dd87cd17a0fb/68747470733a2f2f692e6962622e636f2f63676d6635366b2f323032312d31302d30322d30392d34352d31392e706e67)
> 
> However, you observe this effect with ViT-L/16 vs. ViT-B/16, whereas it was:
> 
> * **not** observed with patch size 16 (ViT-B/16 vs. ViT-S/16) in the table,
> * only observed with patch size 8 (ViT-B/8 vs. ViT-S/8) in the table.
> 
> If confirmed, I suspect this could happen as well with the more expensive runs at patch size 8 (ViT-L/8 vs. ViT-B/8).

Thanks for the information. I also got worse linear probing accuracy.

**Comment by woctezuma** [comment]
> I also got worse linear probing accuracy.

Then we need Mathilde's input, as it could be a matter of tweaking here and there.
